xdsl/frontend/onnx/type.py

```diff
@@ -0,0 +1,36 @@
+from onnx import TensorShapeProto, TypeProto
+
+from xdsl.dialects.builtin import TensorType, f32, f64
+from xdsl.ir import Attribute
+
+ELEM_TYPE = {
+    1: f32,
+    11: f64,
+}
+"Dictionary containing information about the type."
+
+
+def get_elem_type(code: int) -> Attribute:
+    "Function that takes an index as input and returns the corresponding Attribute."
```

Comment:
Can you please change the single quotes to triple quotes? Also, the `Function` seems a bit redundant, could you please remove it from the docstring, and below?
The term `index` isn't clear here, it seems like `ONNX tensor element type code` is a little more accurate/descriptive, do you know what term they use in their documentation to describe this value?